Sources/SWBMacro/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+
+add_library(SWBMacro
+  MacroCondition.swift
+  MacroConditionExpression.swift
+  MacroConditionParameter.swift
+  MacroConditionSet.swift
+  MacroConfigFileDiagnostic.swift
+  MacroConfigFileParser.swift
+  MacroDeclaration.swift
+  MacroEvaluationProgram.swift
+  MacroEvaluationScope.swift
+  MacroExpression.swift
+  MacroExpressionDiagnostic.swift
+  MacroExpressionParsing.swift
+  MacroNamespace.swift
+  MacroType.swift
+  MacroValueAssignmentTable.swift)
+set_target_properties(SWBMacro PROPERTIES
+  Swift_LANGUAGE_VERSION 6)
+target_link_libraries(SWBMacro PUBLIC
+  SWBUtil
+  SwiftDriver
+  TSCBasic)
+
+target_include_directories(SWBMacro PUBLIC
```

Comment:
What is going on here? What bits aren't getting picked up by the linking?

Reply:
I do not understand why these are not getting picked up. There are two cases where I had to explicitly workaround the `target_link_libraries` being insufficient.